src/test/resources/test_data/calculation_breakdown_response/hdc4/crs-2159-prisoner-1.json

```diff
@@ -114,15 +114,10 @@
       "releaseDate": "2025-05-16",
       "unadjustedDate": "2025-12-27"
     },
```

Comment:
I assume this is becase the PRRD is after the CRD so it's now ommitted in this?

Reply:
Yea the CRD isn't in the final dates, so not sure what it was doing here in the first place.